common/coins.go

```diff
@@ -61,13 +61,9 @@ func (a EtherAmount) AsEther() float64 {
 	return res
 }
 
-// ToDecimals returns the amount rounded to a fixed number of decimal places
-func (a EtherAmount) ToDecimals(decimals uint8) float64 {
-	decimalsValue := big.NewInt(0).Exp(big.NewInt(10), big.NewInt(int64(decimals)), nil)
-	wei := big.NewFloat(0).SetInt(a.BigInt())
-	ether := big.NewFloat(0).Quo(wei, big.NewFloat(0).SetInt(decimalsValue))
-	res, _ := ether.Float64()
-	return res
+// AsStandard returns the wei amount as ether
+func (a EtherAmount) AsStandard() float64 {
```

Comment:
We had discussed renaming `EtherAmount` to `WeiAmount` and `MoneroAmount` to `PiconeroAmount` so the type names match the value being stored. Would this PR be a good time to make that change?

Reply:
yep, done!